config/samples/namespaced/verifier/config_v1beta1_verifier_notation_kmprovider.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: config.ratify.deislabs.io/v1beta1
+kind: NamespacedVerifier
+metadata:
+  name: verifier-notation
+spec:
+  name: notation
+  artifactTypes: application/vnd.cncf.notary.signature
+  parameters:
+    verificationCertStores:
+      certs:
+        - kmprovider-akv
```

Comment:
just want to confirm that moving forward these non-namespaced refs will point to a clustered KMP?